### PR TITLE
README elucidation for Ruby beginners

### PR DESCRIPTION
**Pull request by jmsoper**
Added links to referenced tools for beginners, blocked out User model further in Quick Start

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +235,10 @@ def create
   @user = User.create( user_params )
 end
 
+def show
+  @user = User.find( user_params )
+end
+
```
Comment (tute) [inline README.md]: Do we need to add this method here? If it doesn't add to the documentation, I'd drop it, leaving it as it was.

**Comment by tute** [comment]
One small comment, looks good!

**Comment by jmsoper** [comment]
I thought it might be clearer to have it in, since we implement a show view
later in quick start?

**Comment by tute** [comment]
> I thought it might be clearer to have it in, since we implement a show view later in quick start?

That makes sense. What I'd do (and after the conversation with Vadim talking about how we infer the applicatino already exists) is to rename the `Show View` section to `View Helpers`.

**Pull request merged**